en/docs/learn/consume-api/invoke-apis/invoke-apis-using-tools/try-out-using-postman.md

```diff
@@ -0,0 +1,57 @@
+# Try out using Postman.
+
+Let`s see how to download a postman collection.
+
+1.  Sign in to the WSO2 Developer Portal (`https://<hostname>:9443/devportal`) and click an API (e.g. `Pizza` ) to go to the API overview.
+
+    ![({{base_path}}/assets/attachments/postman/postman_api_overview.png)]({{base_path}}/assets/attachments/postman/postman_api_overview.png)
```

Comment:
- Images should be physically added in the `en/docs/assets/img/` folder respective to the section that you are working on. In this case, the images should be in the `en/docs/assets/learn/` section. Add the images directly to this folder without creating a separate folder names "postman".

- Do this for all the images. Make sure that there are no upper case letters in the file names.

- The image definition is wrong. The format of the image definition should be as follows so that if the user clicks on the image he/she will be able to see the zoomed version of the image.
`[![ALT](file-path)](file-path)`

ALT - refers to the text that will appear if the image link breaks and does not appear.

Reply:
Just changing the path of the image in the doc is not enough. You need to change the location of the physical image of the location as well. So move all the images that u added newly to the `en/docs/assets/img/learn/` folder.